src/coverlet.console/Program.cs

```diff
@@ -15,6 +15,42 @@ namespace Coverlet.Console
 {
     class Program
     {
+        /// <summary>
+        /// Exit Codes returned from Coverlet console process.
+        /// </summary>
+        enum CommandExitCodes
+        {
+            /// <summary>
+            /// Indicates successful run of dotnet test without any test failure and coverage percentage above threshold if provided.
+            /// </summary>
+            Success = 0,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test.
+            /// </summary>
+            TestFailed = 1,
+
+            /// <summary>
+            /// Indicates coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            CoverageBelowThreshold = 2,
+
+            /// <summary>
+            /// Indicates test failure by dotnet test and coverage percentage is below given threshold for one or more threshold type.
+            /// </summary>
+            TestFailedAndCoverageBelowThreshold = TestFailed + CoverageBelowThreshold,
+
+            /// <summary>
+            /// Indicates exception occurred during Coverlet process.
+            /// </summary>
+            Exception = 101,
```

Comment:
Why not simply 4,5 for exceptions?

Reply:
If incase tomorrow, we add enhancement to return different exit code for different Threadhold failures, then we can make use of values from 4. If we make Exceptions 4,5 now and then when we that enhancement, order might look. Changing the value then, will raise compatibility issue when people update to new version. 

Idea was 2-100 series represents "after process" failure exit codes.
> 100, represents "during process" failure exit codes.

Reply:
> Changing the value then, will raise compatibility issue when people update to new version.

Yup...as I said I don't know if will be code that verify all these possible "error codes", btw I won't oppose to this if also @tonerdo agree.

Reply:
I will wait for @tonerdo comments before making changes. So, I can submit all code review changes in single commit.

Reply:
Ping @tonerdo ... Waiting for your confirmation...

Reply:
@tonerdo I assume since you had reviewed, you are fine with having different starting values for exceptions.

Reply:
@tonerdo what do you think about > 100 for errors?I don't have a strong opinion...I would say ok...but I don't want fail this choice.

Reply:
ping @tonerdo  ...  what do think about Exception having > 100 exit codes?

Reply:
This is fine

Reply:
Thanks @tonerdo @MarcoRossignoli

Reply:
Thank you and sorry for the delay